targets/mbedos5/mbed-events.lib

```diff
@@ -0,0 +1 @@
+https://github.com/ARMmbed/mbed-events/#a4ba1b08ef90e2b3b6d442696f4d80a8587494e3
```

Comment:
These .lib files definitely need to be in source control, otherwise mbed cli doesn't know what libraries to fetch.